tests/test_reader.py

```diff
@@ -45,6 +45,17 @@ def test_read_getdist():
     mcmc.plot_1d(['x0', 'x1', 'x2', 'x3'])
     plt.close("all")
 
+    os.rename('./tests/example_data/gd.paramnames',
+              './tests/example_data/gd.paramnames_')
+    mcmc = read_getdist('./tests/example_data/gd')
+    os.rename('./tests/example_data/gd.paramnames_',
+              './tests/example_data/gd.paramnames')
```

Comment:
I do wonder whether the equivalent code further down for polychord might have been causing the matplotlib thread-unsafe elements we were experiencing (and if we just made these permanent files, whether we could reinstate parallel test execution).

Reply:
Might be, some other test reading when the needed files have been temporarily renamed... Happy to have some more test files and try bringing back parallel testing.